protocol/x/subaccounts/types/update.go

```diff
@@ -52,11 +52,12 @@ func GetErrorFromUpdateResults(
 }
 
 var updateResultStringMap = map[UpdateResult]string{
-	Success:                        "Success",
-	NewlyUndercollateralized:       "NewlyUndercollateralized",
-	StillUndercollateralized:       "StillUndercollateralized",
-	WithdrawalsAndTransfersBlocked: "WithdrawalsAndTransfersBlocked",
-	UpdateCausedError:              "UpdateCausedError",
+	Success:                               "Success",
+	NewlyUndercollateralized:              "NewlyUndercollateralized",
+	StillUndercollateralized:              "StillUndercollateralized",
+	WithdrawalsAndTransfersBlocked:        "WithdrawalsAndTransfersBlocked",
+	UpdateCausedError:                     "UpdateCausedError",
+	ViolatesIsolatedSubaccountConstraints: "ViolatesIsolatedSubaccountConstraints",
```

Comment:
Is this how the error gets propagated to the client? Do we think we need to provide more info or would this be sufficient?

Reply:
There's several more layers of indirection before this error is sent to the client. I'm planning on looking at the error propagation once integration tests / more E2E testing is added.